spec/fixtures/export/sample_export.xml

```diff
@@ -0,0 +1,205 @@
+<?xml version="1.0" encoding="UTF-8"?>
```

Comment:
@little9 Is this needed for self deposit? - It looks like a FOXML export, so I wouldn't expect to see that in here...